src/views/VideoMania/index.jsx

```
@@ -0,0 +1,100 @@
+            } px-2 py-1 rounded-md  cursor-pointer`}
+            onClick={() => setActiveCategory(category)}
+          >
+            {category.name}
+          </div>
+        ))}
+      </div>
+
+      <div className="mt-10">
+        {videos?.length > 0 &&
+          videos?.map((video) => (
+            <div className="mb-5">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)